pkg/receive/writer_test.go

```diff
@@ -383,7 +383,7 @@ func benchmarkWriter(b *testing.B, labelsNum int, seriesNum int, generateHistogr
 	app, err := m.TenantAppendable("foo")
 	testutil.Ok(b, err)
 
-	w := NewWriter(logger, m)
+	w := NewWriter(logger, m, true)
```

Comment:
Maybe we want to spawn two benchmarks here - one with interning and one without it?

Reply:
Thanks, done 👍